ext/afform/core/afform.php

```diff
@@ -119,7 +119,7 @@ function afform_civicrm_managed(&$entities, $modules) {
         ],
       ];
     }
-    if (!empty($afform['navigation']) && !empty($afform['server_route'])) {
+    if (!empty($afform['navigation'])) {
```

Comment:
@totten maybe this is what you interpreted as being conditional. What's happening here is I'm *removing* the condition.
Previously a nav menu could only be added for Afforms with a server_route defined. That's no longer the case, since now every afform has a route, so nav menu functionality can be made available unconditionally.